int/int_test.go

```diff
@@ -0,0 +1,65 @@
+package int_test
+
+import (
+	"context"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	//nolint:typecheck // this import is used for general prettines, and currently kept
+	. "github.com/openshift/managed-node-metadata-operator/int"
+)
+
+var (
+	i *Integration
+)
+var _ = BeforeSuite(func() {
+	var err error
+	i, err = NewIntegration()
+	Expect(err).NotTo(HaveOccurred())
+	err = i.DisableWebhook()
+	Expect(err).NotTo(HaveOccurred())
+})
+var _ = AfterSuite(func() {
+	i.Shutdown()
+})
+var _ = Describe("Integrationtests", func() {
+
+	Context("When adding a label to a MachineSet", func() {
+		var (
+			TestLabel string
+			TestValue string
+		)
+		Context("When the labels doesn't exist on the Node", func() {
```

Comment:
Small nit: "When the labels don't exist on the Node" or "When the label doesn't exist on the Node"